.github/workflows/bindings-generator.yml

```diff
@@ -0,0 +1,137 @@
+name: Auto-generate Bindings
+
+on:
+  push:
+    branches:
+      - 'generate/aws-lc-sys-*'
```

Comment:
I wonder if it would be better to use a manual trigger with inputs here?

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#providing-inputs

It's a bit more flexible but I guess the branch naming convention works if that's how you plan on always doing it. Or maybe include both methods just in case?

Reply:
The workflow currently assumes that it is free to add commits to the branch it's running against.  I'd prefer to avoid adding complexity around this assumption in the current PR.

We should consider alternative ways to invoke the workflow once we better understand the pain points of this initial approach.